zeus/run/dataloader.py

```diff
@@ -755,7 +755,7 @@ def _start_monitor(self) -> None:
         # Start monitors. Master process starts and records all monitors.
         for index in range(self.world_size):
             monitor = subprocess.Popen(
-                [self.monitor_path, self._power_log_path(index), "0", "100"],
+                [self.monitor_path, self._power_log_path(index), "0", "100", index],
```

Comment:
AFAIK `Popen` accepts a list of `str`s and will error on other data types. Did you check whether this works?

Reply:
My bad. It does not work. I'll do a fix